proposals/proposal-0001.md

```diff
@@ -0,0 +1,223 @@
+most areas where Helm is used.
+
+Once the champion has asked the Helm community as to whether an idea has any
+chance of acceptance, a summary of the proposal should be presented to
+[cncf-helm][]. This gives the author a chance to flesh out the proposal as a
+high quality, well-formatted proposal, and to address initial concerns about the
+proposal.
+
+### Submitting a proposal
+
+Following a discussion on [cncf-helm][], the proposal should be submitted as a
+proposal via a [GitHub pull request][1]. The proposal must be written in
+proposal style as described below, else it will fail review immediately
+(although minor errors may be corrected by the maintainers).
+
+The standard proposal workflow is:
+
+- You, as the proposal author, fork the helm/community repository and create a
+  new proposal
+- Proposals are written in [Markdown][markdown]
+- Push your proposal to your GitHub fork and [submit a pull request][pr].
+
+Once the review process is complete, the project maintainers will approve the
+proposal by commenting on the proposal with a "Looks good to me", or "LGTM" for
+short. (note that this is not the same as accepting your proposal!)
+
+The project maintainers will not unreasonably deny a proposal. Reasons for
+denying proposal status include duplication of effort, being technically
+unsound, not providing proper motivation, not addressing backwards
+compatibility, or not in keeping with Helm's design. The core maintainers can be
+consulted during the approval phase.
+
+Once approved, proposal authors are responsible for collecting community
+feedback on a proposal before submitting a feature that satisfies the proposal
+for review. However, wherever possible, long open-ended discussions on public
+mailing lists should be avoided. Strategies to keep the discussions efficient
+include: setting up a separate meetings for the topic, having the proposal
+author accept private comments in the early design phases, setting up a wiki
+page, etc. proposal authors should use their discretion here.
+
+## What belongs in a successful proposal
+
+Each proposal should have the following parts:
+
+1. Preamble - YAML style headers containing metadata about the proposal,
+   including the proposal number, a short descriptive title, the author names,
+   etc.
+1. Abstract - a short (~200 word) description of the technical issue being
+   addressed.
+1. Specification - The technical specification should describe the syntax and
+   semantics of any new feature. If the proposal describes a standard, the
+   specification should be detailed enough to describe how other community
+   members may re-implement the standard in their product.
+1. Motivation - The motivation is critical for proposals that want to change
+   Helm in a significant way. It should clearly explain why existing features
+   are inadequate to address the problem that the proposal solves. proposal
+   submissions without sufficient motivation may be rejected outright.
```

Comment:
It might be worth mentioning the different profiles/roles and who they are impacted.